MonoidalCategories/gap/BraidedMonoidalCategoriesTest.gi

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# MonoidalCategories: Monoidal and monoidal (co)closed categories
+#
+# Implementations
+#
+
+InstallGlobalFunction( "BraidedMonoidalCategoriesTest",
+        
+    function( cat, assertlevel, a, b )
+
+        local opposite,
+
+              a_op, braiding_a_b, braiding_a_b_op, braiding_inverse_a_b, braiding_inverse_a_b_op, 
+              b_op, braiding_b_a, braiding_b_a_op, braiding_inverse_b_a, braiding_inverse_b_a_op;
+
+        opposite := Opposite( cat );;
+
+        a_op := Opposite( a );;
+        b_op := Opposite( b );;
+
+        if CanCompute( cat, "Braiding" ) then 
+            
+            braiding_a_b := Braiding( a, b );;
+            braiding_b_a := Braiding( b, a );;
+            
+            braiding_inverse_a_b_op := BraidingInverse( a_op, b_op );;
+            braiding_inverse_b_a_op := BraidingInverse( b_op, a_op );;
+            
+            Assert( assertlevel, IsEqualForMorphisms( braiding_inverse_a_b_op, Opposite( braiding_a_b ) ) );
```

Comment:
```suggestion
            Assert( 0, IsEqualForMorphisms( braiding_inverse_a_b_op, Opposite( braiding_a_b ) ) );
```
What's the motivation behind `assertlevel`? I think we can always use 0, because we want the assert to always be checked.